secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -56,6 +58,9 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
         "secondary" -> secondary.storeId,
         "endingDataVersion" -> job.endingDataVersion
       ) {
+        // This dataset _should_ not already be in the whitelist... if it is, this exits.
+        SecondaryWatcherClaimManager.whitelistAdd(job.datasetId)
```

Comment:
Doesn't this have to also be keyed off the secondary instance as well as dataset?